### DISCUSSION TITLE
image-button-response plugin: how to send the buttons to the bottom of the screen while keeping the stimuli on the centre.

### DISCUSSION THREAD

**Discussion by EstelaCG** [jspsych/jsPsych#852] [Q&A] [answered]
Hi all, 

I am developing a choice RT task with an "image-button-response" plugin. I am using this plugin because I would like my participants to be able to do the task not only on their computers but on any tablet/iPad of any size. When coding the task, the stimuli appear in the centre of the screen (so far so good). However, the buttons appear right underneath, with no space between these and the stimuli. How do I send the buttons to the bottom of the screen while maintaining the stimuli in the centre? I have tried assigning a value to the "margin_vertical" plugin parameter, but this automatically sends the stimuli up the screen. I have also tried to add a top margin to the stimuli and play with different values. This helps to bring the stimuli back down a bit, but I am not sure whether it would look centred on different screen sizes. How can I ensure a correct adjustment? Thanks in advance!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @EstelaCG 

You'll need to modify the CSS a bit to get the buttons displayed at the bottom of the screen. The critical DOM element to modify is:

```html
<div id="jspsych-image-button-response-btngroup">
```

This is the element on the page that holds the buttons. You'll want to position this element in a fixed location at the bottom of the screen. I suggest also making the element have `100%` width so that the buttons are centered on the screen.

You can defined a CSS rule in the `<head>` section of the document:

```html
<head>
	<style>
	#jspsych-image-button-response-btngroup {
		position: fixed;
		bottom: 20px; /* margin from the bottom of the screen */
		left: 0px;
		width: 100%;
	}
	</style>
</head>
```

**Reply by EstelaCG** [reply]
It worked! Thank you so much for your help! :)

**Reply by EstelaCG** [reply]
Hi again, @jodeleeuw! I have another question: As well as during the test trials, I am presenting the buttons during the fixation cross trials, for consistency. However, I would like to disable them during this time, but I haven't managed to do so – I have only been able to modify the CSS to create a "disabled" look. Any suggestions? Thanks in advance and apologies if I am making very obvious questions, I am a complete newbie here!

**Reply by jodeleeuw** [reply]
Hi @EstelaCG 

Try using the `visibility: hidden;` CSS rule to hide things on the screen.

**Reply by jodeleeuw** [reply]
Oh sorry, I think I misunderstood. You want to disable the input on the buttons, not disable their appearance. 

You could use the `on_load` event to do this:

```js
var trial = {
	type: 'html-button-response`,
	...
	on_load: function(){
		document.querySelectorAll('button').map(function(btn){
			btn.disabled = true;
		});
	}
}
```

`on_load` is a function that executes after the plugin has loaded. The function above selects all of the button elements on the page and uses the `map()` function to set the `disabled` property to `true`.

**Reply by EstelaCG** [reply]
Super, thanks!
